workflow/01-collect/get-4mers.py.log

```diff
@@ -2,3 +2,18 @@
 2025-08-28 15:39:59.217 | INFO     | __main__:main:64 - N MOLECULES: 200
 2025-08-28 15:39:59.217 | INFO     | __main__:main:66 - Downloading records
 2025-08-28 15:40:22.238 | INFO     | __main__:main:68 - Done, saving
+2025-08-29 16:20:09.347 | INFO     | __main__:main:63 - N RESULTS:   999
+2025-08-29 16:20:09.348 | INFO     | __main__:main:64 - N MOLECULES: 200
+2025-08-29 16:20:09.348 | INFO     | __main__:main:66 - Downloading records
+2025-08-29 16:20:28.683 | INFO     | __main__:main:68 - Done, saving
+2025-08-29 16:40:08.790 | INFO     | __main__:main:65 - N RESULTS:   999
+2025-08-29 16:40:08.790 | INFO     | __main__:main:66 - N MOLECULES: 200
+2025-08-29 16:40:08.791 | INFO     | __main__:main:68 - Downloading records
+2025-08-29 16:52:50.969 | INFO     | __main__:main:65 - N RESULTS:   999
+2025-08-29 16:52:50.969 | INFO     | __main__:main:66 - N MOLECULES: 200
+2025-08-29 16:52:50.969 | INFO     | __main__:main:68 - Downloading records
+2025-08-29 17:02:40.013 | INFO     | __main__:main:73 - Done, saving
+2025-08-29 17:09:57.082 | INFO     | __main__:main:65 - N RESULTS:   999
+2025-08-29 17:09:57.082 | INFO     | __main__:main:66 - N MOLECULES: 200
+2025-08-29 17:09:57.082 | INFO     | __main__:main:68 - Downloading records
+2025-08-29 17:19:06.260 | INFO     | __main__:main:73 - Done, saving
```

Comment:
Just wondering, does this mean you ran the script 4 times, or something happened 4 times in the script?

Reply:
Mmm that'll be because I forgot to delete it between runs. I'll delete and run it again so there's a clean log.